src/ansys/fluent/core/solver/monitors_manager.py

```diff
@@ -94,7 +93,7 @@ def get_monitor_set_data(
         Returns
         -------
         Tuple[np.array, Dict[str, np.array]]
-            Tuple containing a numpy array of x-axis values and a dictionary of monitor names
+            Tuple containing a numpy array of x-axis values, a dictionary of monitor names,
```

Comment:
Based on the type hint above (as well as the implementation), I believe that the implementer intended to say, "Tuple containing two elements: a numpy array of x-axis values and a dictionary associating monitor names of type str to numpy arrays of y-axis values".